### PR TITLE
fixed issue #3182 - alphabetized dependencies in quick tutorial and a…

### PR DESCRIPTION
**Pull request by radurevutchi**
fixed issue #3182 

alphabetized dependencies in setup.py files in quick_tutorial and aligned the all of the emphasize-lines to the new order.

### CODE REVIEW

**Comment by cewing** [comment]
@stevepiercy I'm working with @radurevutchi at pycon on this and other issues.  Feel free to yell at me if I've helped him poorly.

**Comment by bertjwregeer** [comment]
@goodwillcoding please don't merge without verifying with @stevepiercy, this may also need to get back ported to older versions if it makes sense to do so.

**Comment by goodwillcoding** [comment]
@stevepiercy @x58 ... it seemed simple enough for new folks in PyCon sprint, I can always revert it if it is a problem just let me know.

**Comment by bertjwregeer** [comment]
@goodwillcoding nah, no need to revert. Just a heads up for next time.

**Comment by goodwillcoding** [comment]
@bertjwregeer noted

File: docs/quick_tutorial/authorization/setup.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
 from setuptools import setup
 
 requires = [
+    'bcrypt',
     'pyramid',
-    'waitress',
     'pyramid_chameleon',
-    'bcrypt'
+    'waitress',
+
```
Comment (stevepiercy) [inline docs/quick_tutorial/authorization/setup.py]: Remove extra line.

File: docs/quick_tutorial/json/setup.py [hunk 2]
```diff
@@ -2,8 +2,9 @@
 
 requires = [
     'pyramid',
+    'pyramid_chameleon',
     'waitress',
-    'pyramid_chameleon'
+
```
Comment (stevepiercy) [inline docs/quick_tutorial/json/setup.py]: Remove extra line

File: docs/quick_tutorial/logging/setup.py [hunk 3]
```diff
@@ -2,8 +2,9 @@
 
 requires = [
     'pyramid',
+    'pyramid_chameleon',
     'waitress',
-    'pyramid_chameleon'
+
```
Comment (stevepiercy) [inline docs/quick_tutorial/logging/setup.py]: Remove extra line

File: docs/quick_tutorial/more_view_classes/setup.py [hunk 4]
```diff
@@ -2,8 +2,9 @@
 
 requires = [
     'pyramid',
+    'pyramid_chameleon',
     'waitress',
-    'pyramid_chameleon'
+
```
Comment (stevepiercy) [inline docs/quick_tutorial/more_view_classes/setup.py]: Remove extra line

File: docs/quick_tutorial/retail_forms/setup.py [hunk 5]
```diff
@@ -1,10 +1,11 @@
 from setuptools import setup
 
 requires = [
+    'deform',
     'pyramid',
-    'waitress',
     'pyramid_chameleon',
-    'deform'
+    'waitress',
+
```
Comment (stevepiercy) [inline docs/quick_tutorial/retail_forms/setup.py]: Remove extra line

File: docs/quick_tutorial/sessions/setup.py [hunk 6]
```diff
@@ -2,8 +2,9 @@
 
 requires = [
     'pyramid',
+    'pyramid_chameleon',
     'waitress',
-    'pyramid_chameleon'
+
```
Comment (stevepiercy) [inline docs/quick_tutorial/sessions/setup.py]: Remove extra line

File: docs/quick_tutorial/static_assets/setup.py [hunk 7]
```diff
@@ -2,8 +2,9 @@
 
 requires = [
     'pyramid',
+    'pyramid_chameleon',
     'waitress',
-    'pyramid_chameleon'
+
```
Comment (stevepiercy) [inline docs/quick_tutorial/static_assets/setup.py]: Remove extra line

File: docs/quick_tutorial/templating/setup.py [hunk 8]
```diff
@@ -2,8 +2,9 @@
 
 requires = [
     'pyramid',
-    'waitress',
     'pyramid_chameleon',
+    'waitress',
+
```
Comment (stevepiercy) [inline docs/quick_tutorial/templating/setup.py]: Remove extra line

File: docs/quick_tutorial/view_classes/setup.py [hunk 9]
```diff
@@ -2,8 +2,9 @@
 
 requires = [
     'pyramid',
+    'pyramid_chameleon',
     'waitress',
-    'pyramid_chameleon'
+
```
Comment (stevepiercy) [inline docs/quick_tutorial/view_classes/setup.py]: Remove extra line

**Comment by stevepiercy** [comment]
Actually, it needs to be updated because there are extra lines added in a numerous places that should not have been added. So either revert this PR or open another PR to remove the extra lines.

Additionally, I want to double-check that no references to `emphasize-lines` were missed or become misaligned.

I'm fine with merging PRs, but only after they have been carefully reviewed.

**Comment by radurevutchi** [comment]
Hi Steve and Cris,

I have removed the extra blank lines from the lists of required dependencies so that it matches with the original formatting. This is all viewable in the new PR.

- Radu

**Pull request merged**